### PR TITLE
Src 6413 create a collision model for the rack

### PR DESCRIPTION
**Pull request by maxzieba**
## Proposed changes

User story: https://shadowrobot.atlassian.net/browse/SRC-6413
Adding server rack model.

## Checklist

If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **ALL** boxes are checked **BEFORE** the PR is merged.

- [X] Manually tested that added code works as intended (if any functional/runnable code is added).
- [ ] Added automated tests (if a new class is added (Python or C++), interface of that class must be unit tested).
- [ ] Tested on real hardware (if the changed or added code is used with the real hardware).
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc.)

### CODE REVIEW

**Review by giusebar — CHANGES_REQUESTED** [review]
Missing scene file

File: sr_description_common/models/server_rack/model.sdf [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        <geometry>
+          <box>
+            <size>0.075 1.2 1.9</size>
+          </box>
+        </geometry>
+      </visual>
+    </link>
+
+    <link name="server_wall_back">
+
```
Comment (beatrizleon) [inline sr_description_common/models/server_rack/model.sdf]: remove this empty line

File: sr_description_common/models/server_rack/model.sdf [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        <geometry>
+          <box>
+            <size>0.45 0.075 1.9</size>
+          </box>
+        </geometry>
+      </visual>
+    </link>
+
+    <link name="server_wall_top">
+
```
Comment (beatrizleon) [inline sr_description_common/models/server_rack/model.sdf]: here again

File: sr_description_common/models/server_rack/model.sdf [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        <geometry>
+          <box>
+            <size>0.6 1.2 0.075</size>
+          </box>
+        </geometry>
+      </visual>
+    </link>
+
+</model>
+</sdf>
```
Comment (beatrizleon) [inline sr_description_common/models/server_rack/model.sdf]: add empty line at end of file

Comment (maxzieba) [reply sr_description_common/models/server_rack/model.sdf]: Added

File: sr_description_common/models/server_rack/model.sdf (hunk 1 above)
Comment (maxzieba) [reply sr_description_common/models/server_rack/model.sdf]: Removed

File: sr_description_common/models/server_rack/model.sdf (hunk 2 above)
Comment (maxzieba) [reply sr_description_common/models/server_rack/model.sdf]: Removed

**Review by giusebar — APPROVED** [review]

**Review by georgiablanco — APPROVED** [review]

**Pull request merged**